<?xml version="1.0"?>
<launch>

  <arg name="joy_dev" default="/dev/input/js0" />
  <arg name="joystick" default="true" />
  <arg name="config" default="$(find subt_example)/config/logitech_config.yaml"/>

  <rosparam command="load" file="$(arg config)" />
  <param name="joy_node/dev" value="$(arg joy_dev)" />

  <node pkg="joy" type="joy_node" name="joy_node" />

  <node pkg="subt_example" type="teleop_node" name="teleop_twist_joy"/>

</launch>
